### ISSUE TITLE
503 when logging in

### ISSUE THREAD

**Issue by jmordica** [sipcapture/homer-app#65]
I'm running the quick install Homer 7 and when attempting to login with admin/sipcapture i'm getting a 503 from the api service. I'm not able to connect to the running container to see what the logs look like. Any suggestions or known issues?

**Comment by lmangani** [comment]
Hi Joe,

There's really no quick install for H7 only containers. Could you confirm what instructions you followed?

**Comment by jmordica** [comment]
Yea sorry it was the docket containers. hep7-influx. The only other thing I did was use nginx as a reverse proxy to port 9060. But I tried it without nginx and got the same 503 when trying to auth. The hep5-influx works fine and I can auth with admin/test123 but it seems to use a different backend api.

**Comment by lmangani** [comment]
Hi Joe,

Indeed, the API between 5 and 7 is entirely different. The latest containers should all work great as-is unless there are errors intializing. Specifically this one should match your setup:

https://github.com/sipcapture/homer7-docker/tree/master/heplify-server/hom7-hep-influx

**Comment by jmordica** [comment]
Yep so all I did is docker-compose up -d and verified that all the containers started properly. I guess I will need to attach to the container running the api/nodejs server and figure out what is going on with it.

**Comment by lmangani** [comment]
`docker logs --follow homer-webapp` should do to get an idea

**Comment by jmordica** [comment]
Everything works fine with hom7-hep-prom-graf

**Comment by lmangani** [comment]
@jmordica could you pull the latest `sipcapture/homer-app:master` container and try again?

**Comment by areski** [comment]
@lmangani the last container for `homer-app` have an issue with the migration, see https://github.com/sipcapture/homer-app/commit/929a0454fb007bbec3f46141c3d962619820aad9#diff-528c0c313081bc1040c1262dce60b32c 

It would be great if you could push an update!

**Comment by lmangani** [comment]
@areski could you open a separate issue for that with the case details? We'll look in parallel

**Comment by areski** [comment]
Surely, sorry for hijacking this issue!

**Comment by lmangani** [comment]
Closing in preparation for the 7.7 Release. Please re-test your issue using the 7.7. branch or Compose containers and feel free to reopen w/ fresh details if needed. Thanks!
